src/test/java/io/strimzi/kafka/bridge/http/ProducerIT.java

```diff
@@ -648,7 +648,7 @@ void sendToOneStringPartitionTest(VertxTestContext context) throws InterruptedEx
 
         producerService()
             .sendRecordsToPartitionRequest(topic, partition, root, BridgeContentType.KAFKA_JSON_JSON)
-            .sendJsonObject(root, verifyBadRequest(context, "Validation error on: partitionid - Value is not a valid number"));
+            .sendJsonObject(root, verifyBadRequest(context, "Validation error on: partitionid - [Bad Request] Parsing error for parameter partitionid in location PATH: java.lang.NumberFormatException: For input string: \"" + partition + "\""));
```

Comment:
```suggestion
            .sendJsonObject(root, verifyBadRequest(context, "Validation error on: partitionid - [Bad Request] Parsing error for the partitionid parameter in location PATH: java.lang.NumberFormatException: For input string: \"" + partition + "\""));
```

Reply:
These are messages from the vertx library.